components/generic/CiteAttribution.vue

```diff
@@ -1,5 +1,61 @@
 <template>
-  <cite>
+  <cite v-if="extended">
+    <p v-if="name !== null">{{ $t('attribution.title') }}
```

Comment:
`!== null` is redundant. If the attributes are null this'll evaluate to false: `<p v-if="name">`

Same for the other values url, creator and provider

Reply:
I have removed `!== null` I added it because previously I found that the creator tag was rendering without a value on http://localhost:3000/en/exhibitions/edible-plants-from-the-americas
but testing again with just `v-if="creator"` it is not rendering anymore.